entities/gamecharacters/Player/PlayerCharacter.cs

```diff
@@ -54,7 +54,10 @@ public class PlayerCharacter : GameCharacter {
 		else if (_noclipping && Input.IsActionJustReleased("debug_elev_noclip")) _noclipping = false;
 		ProcessDebugElevChanges();
 		SetAnimation(inputs);
-		Velocity = inputs.Direction * MoveSpeed;
+		var finalSpeed = MoveSpeed;
+		if (Input.IsActionPressed("run"))
+		{ finalSpeed += 50; };
+		Velocity = inputs.Direction * finalSpeed;
```

Comment:
Suggestion: Add RunSpeed export to top of file
```diff
+[Export]
+int RunSpeed = 50;
```
Then use this more sane logic:
```suggestion
		if (Input.IsActionPressed("run")) Velocity = inputs.Direction * RunSpeed;
		else Velocity = inputs.Direction * MoveSpeed;
```

Reply:
And honestly it might even be more sensible to implement case logic here for, say, crawlspeed, limpspeed, and other things we might want to add one day